src/api/logs.ts

```diff
@@ -0,0 +1,29 @@
+import express from 'express'
+import fs from 'fs'
+import environment from '../helpers/environment'
+import { getMappingByDomain } from '../lib/data'
+
+const logsRouter = express.Router()
+const { isProduction } = environment
+
+logsRouter.get('/err/:domain', (req, res) => {
+  const { domain } = req.params
+  const { fullDomain } = getMappingByDomain(domain)
+
+  if (!isProduction()) return res.send('OK')
```

Comment:
Why not get the logs in dev? I don't have context but, a comment in the code might be nice for future devs.

Reply:
I updated the codebase to allow for access of logs in dev.  From the current codebase with c0d3.com, the proxy server will not run when running in dev. The user will have to navigate to the port on the server to access their apps.

Reply:
@joshgreenwell `logs` are directly tied to `root pm2`, so one should be running in production flag when testing these to simulate production as much as possible.

At this point I don't see any need other than prod vs non-prod.

Usually `test` vs `stage` vs `sandbox` environment comes into importance when you start dealing with databases and billing.